.github/workflows/integration_test.yaml

```diff
@@ -34,8 +34,17 @@ jobs:
       - name: Install tox
         run: python3 -m pip install tox
 
-      - name: Build docker image
-        run: tox -e test-build
+      - name: Install tox, skopeo and docker
```

Comment:
Any reason this isn't using `operator-workflows`?

Reply:
I think it is because of Devstack usage for testing. I'll take a look anyway and make adjustments if needed.